doc/rdoc/markup_reference.rb

```diff
@@ -75,14 +88,16 @@
 #   #
 #   # You'll love it.
 #
-# Rendered HTML:
+# >>> Rendered HTML >>>
 #
 # \RDoc produces HTML and command-line documentation for Ruby projects.
 # \RDoc includes the rdoc and ri tools for generating and displaying
 # documentation from the command-line.
 #
 # You'll love it.
 #
+# <<<<<<
```

Comment:
Isn't BLOCKQUOTE better?

```suggestion
# Rendered HTML:
#
# >>>
#   \RDoc produces HTML and command-line documentation for Ruby projects.
#   \RDoc includes the rdoc and ri tools for generating and displaying
#   documentation from the command-line.
#
#   You'll love it.
```

Reply:
Rendered using BLOCKQUOTE: <img width="553" alt="image" src="https://user-images.githubusercontent.com/16700/181156923-7d83cad9-7d3c-4964-b870-b92d1af67a0d.png">

Reply:
Thanks, @nobu.  I was hoping for suggested better delimitation.  I'm not sure that all markup will be honored inside a blockquote, but I'll look into it.

Another idea I had was to put ```Rendered HTML```, then the HTML preceded and followed by horizontal rules.  That becomes a problem when I need to show a rendered HR. but maybe that's a small price to pay.

Reply:
I see that list items, HR, and verbatim text (including code blocks) survive in a blockquote.  Don't know what else does, or not.  I'll go ahead and switch over.